frontend/server/src/Controllers/User.php

```diff
@@ -3374,7 +3376,7 @@ public static function getRankDetailsForSmarty(\OmegaUp\Request $r) {
      * @omegaup-request-param mixed $category
      * @omegaup-request-param mixed $date
      *
-     * @return array{smartyProperties: array{payload: array{coderOfTheMonthData: array{all: array{birth_date: int|null, classname: string, country: string, country_id: null|string, email: null|string, gender: null|string, graduation_date: int|null, gravatar_92: string, hide_problem_tags: bool|null, is_private: bool, locale: string, name: null|string, preferred_language: null|string, scholar_degree: null|string, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: null|string, verified: bool}|null, female: array{birth_date: int|null, classname: string, country: string, country_id: null|string, email: null|string, gender: null|string, graduation_date: int|null, gravatar_92: string, hide_problem_tags: bool|null, is_private: bool, locale: string, name: null|string, preferred_language: null|string, scholar_degree: null|string, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: null|string, verified: bool}|null}, currentUserInfo: array{username?: string}, enableSocialMediaResources: bool, userRank: list<CoderOfTheMonth>, runsChartPayload: array{date: list<string>, total: list<int>}, schoolOfTheMonthData: array{country_id: null|string, country: null|string, name: string, school_id: int, state: null|string}|null, schoolRank: list<array{name: string, ranking: int, school_id: int, school_of_the_month_id: int, score: float}>, upcomingContests: array{number_of_results: int, results: list<array{alias: string, title: string}>}}}, supportsBootstrap4: bool, template: string}
+     * @return array{smartyProperties: array{payload: IndexPayload}, supportsBootstrap4: bool, template: string}
```

Comment:
```suggestion
     * @return array{smartyProperties: array{indexPayload: IndexPayload}, supportsBootstrap4: bool, template: string}
```

esto mejora la consistencia, pero no se si mejore o empeore los planes de hacer la plantilla única y uniforme. si sientes que esto lo empeoraría, siéntete libre de ignorarlo.

Reply:
Idealmente, el payload para el componente principal a ser desplegado, debe ser uno solo y llamarse `payload`. Así que prefiero no hacer ese cambio.